Include/fileutils.h

```diff
@@ -18,6 +18,8 @@ PyAPI_FUNC(char*) _Py_EncodeLocaleRaw(
     size_t *error_pos);
 #endif
 
+PyAPI_FUNC(void) _Py_closerange(int first, int last);
```

Comment:
Since it's a private function, I would prefer to define it in Include/internal/pycore_fileutils.h.

If you consider that it's a really important function, make it public (remove "_" prefix). I would prefer to declare it as an internal function.

Reply:
I had originally started going that route, but either subprocess of posixmodule (I want to say subprocess) doesn't get built in the needed context for that (from memory, that header required some kind of pycore define).